lms/static/sass/base/_base.scss

```diff
@@ -251,7 +251,7 @@ mark {
 
 // UI - is hidden
 .is-hidden {
-  display: none;
+  display: none !important;
```

Comment:
Yeah, I'm often the one to say, "just make the selector more specific!", but I don't think that applies here, as this is mean to be fairly general.

:+1: